<!DOCTYPE html>
<html lang="en">
    <!-- title -->




<!-- keywords -->




<head><meta name="generator" content="Hexo 3.8.0">
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
    <meta name="author" content="Hua">
    <meta name="renderer" content="webkit">
    <meta name="copyright" content="Hua">
    
    <meta name="keywords" content="hexo,hexo-theme,hexo-blog">
    
    <meta name="description" content="">
    <meta http-equiv="Cache-control" content="no-cache">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <title>Calapin3详析 · Hua的试验记录</title>
    <style type="text/css">
    @font-face {
        font-family: 'Oswald-Regular';
        src: url("/font/Oswald-Regular.ttf");
    }

    body {
        margin: 0;
    }

    header,
    footer,
    .back-top,
    .sidebar,
    .container,
    .site-intro-meta,
    .toc-wrapper {
        display: none;
    }

    .site-intro {
        position: relative;
        z-index: 3;
        width: 100%;
        /* height: 50vh; */
        overflow: hidden;
    }

    .site-intro-placeholder {
        position: absolute;
        z-index: -2;
        top: 0;
        left: 0;
        width: calc(100% + 300px);
        height: 100%;
        background: repeating-linear-gradient(-45deg, #444 0, #444 80px, #333 80px, #333 160px);
        background-position: center center;
        transform: translate3d(-226px, 0, 0);
        animation: gradient-move 2.5s ease-out 0s infinite;
    }

    @keyframes gradient-move {
        0% {
            transform: translate3d(-226px, 0, 0);
        }
        100% {
            transform: translate3d(0, 0, 0);
        }
    }

</style>

    <link rel="preload" href="/css/style.css?v=20180824" as="style" onload="this.onload=null;this.rel='stylesheet'">
    <link rel="stylesheet" href="/css/mobile.css?v=20180824" media="(max-width: 980px)">
    
    <link rel="preload" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
    
    <!-- /*! loadCSS. [c]2017 Filament Group, Inc. MIT License */
/* This file is meant as a standalone workflow for
- testing support for link[rel=preload]
- enabling async CSS loading in browsers that do not support rel=preload
- applying rel preload css once loaded, whether supported or not.
*/ -->
<script>
(function( w ){
	"use strict";
	// rel=preload support test
	if( !w.loadCSS ){
		w.loadCSS = function(){};
	}
	// define on the loadCSS obj
	var rp = loadCSS.relpreload = {};
	// rel=preload feature support test
	// runs once and returns a function for compat purposes
	rp.support = (function(){
		var ret;
		try {
			ret = w.document.createElement( "link" ).relList.supports( "preload" );
		} catch (e) {
			ret = false;
		}
		return function(){
			return ret;
		};
	})();

	// if preload isn't supported, get an asynchronous load by using a non-matching media attribute
	// then change that media back to its intended value on load
	rp.bindMediaToggle = function( link ){
		// remember existing media attr for ultimate state, or default to 'all'
		var finalMedia = link.media || "all";

		function enableStylesheet(){
			link.media = finalMedia;
		}

		// bind load handlers to enable media
		if( link.addEventListener ){
			link.addEventListener( "load", enableStylesheet );
		} else if( link.attachEvent ){
			link.attachEvent( "onload", enableStylesheet );
		}

		// Set rel and non-applicable media type to start an async request
		// note: timeout allows this to happen async to let rendering continue in IE
		setTimeout(function(){
			link.rel = "stylesheet";
			link.media = "only x";
		});
		// also enable media after 3 seconds,
		// which will catch very old browsers (android 2.x, old firefox) that don't support onload on link
		setTimeout( enableStylesheet, 3000 );
	};

	// loop through link elements in DOM
	rp.poly = function(){
		// double check this to prevent external calls from running
		if( rp.support() ){
			return;
		}
		var links = w.document.getElementsByTagName( "link" );
		for( var i = 0; i < links.length; i++ ){
			var link = links[ i ];
			// qualify links to those with rel=preload and as=style attrs
			if( link.rel === "preload" && link.getAttribute( "as" ) === "style" && !link.getAttribute( "data-loadcss" ) ){
				// prevent rerunning on link
				link.setAttribute( "data-loadcss", true );
				// bind listeners to toggle media back
				rp.bindMediaToggle( link );
			}
		}
	};

	// if unsupported, run the polyfill
	if( !rp.support() ){
		// run once at least
		rp.poly();

		// rerun poly on an interval until onload
		var run = w.setInterval( rp.poly, 500 );
		if( w.addEventListener ){
			w.addEventListener( "load", function(){
				rp.poly();
				w.clearInterval( run );
			} );
		} else if( w.attachEvent ){
			w.attachEvent( "onload", function(){
				rp.poly();
				w.clearInterval( run );
			} );
		}
	}


	// commonjs
	if( typeof exports !== "undefined" ){
		exports.loadCSS = loadCSS;
	}
	else {
		w.loadCSS = loadCSS;
	}
}( typeof global !== "undefined" ? global : this ) );
</script>

    <link rel="icon" href="/assets/favicon.ico">
    <link rel="preload" href="https://cdn.jsdelivr.net/npm/webfontloader@1.6.28/webfontloader.min.js" as="script">
    <link rel="preload" href="https://cdn.jsdelivr.net/npm/jquery@3.3.1/dist/jquery.min.js" as="script">
    <link rel="preload" href="/scripts/main.js" as="script">
    <link rel="preload" as="font" href="/font/Oswald-Regular.ttf" crossorigin="">
    <link rel="preload" as="font" href="https://at.alicdn.com/t/font_327081_1dta1rlogw17zaor.woff" crossorigin="">
    
        <!-- algolia -->
        <script>
            
            var hits = JSON.parse('{"per_page":10}')
            var labels = JSON.parse('{"input_placeholder":"Search for Posts","hits_empty":"We did not find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}')

            var algolia = {
                applicationID: 'ZKGZPC2HRL',
                apiKey: 'd14aa6ef670d284260dc375c7e565f5e',
                indexName: 'hexo_search',
                hits: hits,
                labels: labels
            }
        </script>
    
    <!-- fancybox -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" defer></script>
    <!-- 百度统计  -->
    
    <!-- 谷歌统计  -->
    
</head>

    
        <body class="post-body">
    
    
<header class="header">

    <div class="read-progress"></div>
    <div class="header-sidebar-menu">&#xe775;</div>
    <!-- post页的toggle banner  -->
    
    <div class="banner">
            <div class="blog-title">
                <a href="/">Hua的试验记录</a>
            </div>
            <div class="post-title">
                <a href="#" class="post-name">Calapin3详析</a>
            </div>
    </div>
    
    <a class="home-link" href="/">Hua的试验记录</a>
</header>
    <div class="wrapper">
        <div class="site-intro" style="







height:50vh;
">
    
    <!-- 主页  -->
    
    
    <!-- 404页  -->
            
    <div class="site-intro-placeholder"></div>
    <div class="site-intro-img" style="background-image: url(//raw.githubusercontent.com/Cpresident/egee/master/20200205111526.png)"></div>
    <div class="site-intro-meta">
        <!-- 标题  -->
        <h1 class="intro-title">
            <!-- 主页  -->
            
            Calapin3详析
            <!-- 404 -->
            
        </h1>
        <!-- 副标题 -->
        <p class="intro-subtitle">
            <!-- 主页副标题  -->
            
            
            <!-- 404 -->
            
        </p>
        <!-- 文章页meta -->
        
            <div class="post-intros">
                <!-- 文章页标签  -->
                
                    <div class="post-intro-tags">
    
        <a class="post-tag" href="javascript:void(0);" data-tags="LGMD">LGMD</a>
    
</div>
                
                
                    <div class="post-intro-read">
                        <span>Word count: <span class="post-count word-count">3.6k</span>Reading time: <span class="post-count reading-time">21 min</span></span>
                    </div>
                
                <div class="post-intro-meta">
                    <span class="post-intro-calander iconfont-archer">&#xe676;</span>
                    <span class="post-intro-time">2020/02/05</span>
                    
                    <span id="busuanzi_container_page_pv" class="busuanzi-pv">
                        <span class="iconfont-archer">&#xe602;</span>
                        <span id="busuanzi_value_page_pv"></span>
                    </span>
                    
                    <span class="shareWrapper">
                        <span class="iconfont-archer shareIcon">&#xe71d;</span>
                        <span class="shareText">Share</span>
                        <ul class="shareList">
                            <li class="iconfont-archer share-qr" data-type="qr">&#xe75b;
                                <div class="share-qrcode"></div>
                            </li>
                            <li class="iconfont-archer" data-type="weibo">&#xe619;</li>
                            <li class="iconfont-archer" data-type="qzone">&#xe62e;</li>
                            <li class="iconfont-archer" data-type="twitter">&#xe634;</li>
                            <li class="iconfont-archer" data-type="facebook">&#xe67a;</li>
                        </ul>
                    </span>
                </div>
            </div>
        
    </div>
</div>
        <script>
 
  // get user agent
  var browser = {
    versions: function () {
      var u = window.navigator.userAgent;
      return {
        userAgent: u,
        trident: u.indexOf('Trident') > -1, //IE内核
        presto: u.indexOf('Presto') > -1, //opera内核
        webKit: u.indexOf('AppleWebKit') > -1, //苹果、谷歌内核
        gecko: u.indexOf('Gecko') > -1 && u.indexOf('KHTML') == -1, //火狐内核
        mobile: !!u.match(/AppleWebKit.*Mobile.*/), //是否为移动终端
        ios: !!u.match(/\(i[^;]+;( U;)? CPU.+Mac OS X/), //ios终端
        android: u.indexOf('Android') > -1 || u.indexOf('Linux') > -1, //android终端或者uc浏览器
        iPhone: u.indexOf('iPhone') > -1 || u.indexOf('Mac') > -1, //是否为iPhone或者安卓QQ浏览器
        iPad: u.indexOf('iPad') > -1, //是否为iPad
        webApp: u.indexOf('Safari') == -1, //是否为web应用程序，没有头部与底部
        weixin: u.indexOf('MicroMessenger') == -1, //是否为微信浏览器
        uc: u.indexOf('UCBrowser') > -1 //是否为android下的UC浏览器
      };
    }()
  }
  console.log("userAgent:" + browser.versions.userAgent);

  // callback
  function fontLoaded() {
    console.log('font loaded');
    if (document.getElementsByClassName('site-intro-meta')) {
      document.getElementsByClassName('intro-title')[0].classList.add('intro-fade-in');
      document.getElementsByClassName('intro-subtitle')[0].classList.add('intro-fade-in');
      var postIntros = document.getElementsByClassName('post-intros')[0]
      if (postIntros) {
        postIntros.classList.add('post-fade-in');
      }
    }
  }

  // UC不支持跨域，所以直接显示
  function asyncCb(){
    if (browser.versions.uc) {
      console.log("UCBrowser");
      fontLoaded();
    } else {
      WebFont.load({
        custom: {
          families: ['Oswald-Regular']
        },
        loading: function () {  //所有字体开始加载
          // console.log('loading');
        },
        active: function () {  //所有字体已渲染
          fontLoaded();
        },
        inactive: function () { //字体预加载失败，无效字体或浏览器不支持加载
          console.log('inactive: timeout');
          fontLoaded();
        },
        timeout: 5000 // Set the timeout to two seconds
      });
    }
  }

  function asyncErr(){
    console.warn('script load from CDN failed, will load local script')
  }

  // load webfont-loader async, and add callback function
  function async(u, cb, err) {
    var d = document, t = 'script',
      o = d.createElement(t),
      s = d.getElementsByTagName(t)[0];
    o.src = u;
    if (cb) { o.addEventListener('load', function (e) { cb(null, e); }, false); }
    if (err) { o.addEventListener('error', function (e) { err(null, e); }, false); }
    s.parentNode.insertBefore(o, s);
  }

  var asyncLoadWithFallBack = function(arr, success, reject) {
      var currReject = function(){
        reject()
        arr.shift()
        if(arr.length)
          async(arr[0], success, currReject)
        }

      async(arr[0], success, currReject)
  }

  asyncLoadWithFallBack([
    "https://cdn.jsdelivr.net/npm/webfontloader@1.6.28/webfontloader.min.js", 
    "https://cdn.bootcss.com/webfont/1.6.28/webfontloader.js",
    "/lib/webfontloader.min.js"
  ], asyncCb, asyncErr)
</script>        
        <img class="loading" src="/assets/loading.svg" style="display: block; margin: 6rem auto 0 auto; width: 6rem; height: 6rem;" />
        <div class="container container-unloaded">
            <main class="main post-page">
    <article class="article-entry">
        <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/hint.css/2.4.1/hint.min.css"><h2 id="Calpain3基因及其家族"><a href="#Calpain3基因及其家族" class="headerlink" title="Calpain3基因及其家族"></a>Calpain3基因及其家族</h2><p><strong>【CAPN基因家族】</strong></p>
<ul>
<li>NS，IS1，IS2为CAPN3所独有</li>
</ul>
<p><img src="https://raw.githubusercontent.com/Cpresident/egee/master/20200205103845.png" alt=""></p>
<p>Fig. 1. Schematic showing the structure of the human calpains. The calpains shown in red are predominantly expressed in specific tissues or organs; those in black are more widely expressed. GR, Gly-rich domain; C2, protein kinase C conserved domain 2; MIT, microtubule interacting and transport motif; Zn, zinc-finger motif; SOL, <em>small optic lobes</em> product homology domain; IQ, calmodulin (CaM)-interacting motif.</p>
<table>
<thead>
<tr>
<th style="text-align:center">片段</th>
<th style="text-align:center">功能</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center">NS</td>
<td style="text-align:center">未知</td>
</tr>
<tr>
<td style="text-align:center">PC1</td>
<td style="text-align:center">proteolytic</td>
</tr>
<tr>
<td style="text-align:center">IS1</td>
<td style="text-align:center">autolysis</td>
</tr>
<tr>
<td style="text-align:center">PC2</td>
<td style="text-align:center">proteolytic</td>
</tr>
<tr>
<td style="text-align:center">CBSW</td>
<td style="text-align:center">substrate recognition</td>
</tr>
<tr>
<td style="text-align:center">IS2</td>
<td style="text-align:center">磷酸化后固定在titin等上，激活后autolysis</td>
</tr>
<tr>
<td style="text-align:center">PEF</td>
<td style="text-align:center">substrate recognition，形成二聚体</td>
</tr>
</tbody>
</table>
<blockquote>
<p>2.1. Discovery of the calpain family<br>In January of 1964, calpain (calcium-dependent papain-like protease; Clan CA, family C02; EC3.4.22.17) was described for the first time by Gordon Guroff [35]. The primary structure of the calpain catalytic subunit was determined in 1984 [37], revealing a multi-domain structure consisting of a cysteine protease domain, now called a Calpain-type cysteine Protease conserved (CysPc) domain [38], [39], [40], a Calpain-type Beta-SandWich (CBSW) domain, previously called a C2-domain-like (C2L) domain, and a Penta E-F hand (PEF) [41] domain (Fig. 1). The CysPc domain is composed of two Protease Core subdomains (PC1 and PC2). Until 1989, most calpain studies focused on calpain-1 (previously called μ-calpain, μCANP, or calpain-I) and calpain-2 (m-calpain, mCANP, or calpain-II). Thus, these calpains are called “conventional” calpains. The terms calpain-1 and calpain-2 are now reserved for the active enzymes, which are heterodimers consisting of a larger ca. 80-kDa catalytic subunit, either CAPN1 (previously called μCL or μ80K) or CAPN2 (mCL or m80K), and a smaller ca. 28-kDa regulatory subunit, CAPNS1 (30K) (Fig. 1). </p>
<p>CAPN3 has three regions that are not found in CAPN1 or CAPN2. These regions are at its N-terminus (NS), within the PC2 domain (IS1), and between the CBSW and PEF domains (IS2) (Fig. 1).</p>
</blockquote>
<p><strong>【CAPN3可变剪接】</strong></p>
<p><img src="https://raw.githubusercontent.com/Cpresident/egee/master/20200205104002.png" alt=""></p>
<p>Fig. 3. Representative CAPN3 isoforms generated by alternative splicing of <em>CAPN3/Capn3</em>. See <a href="https://www.sciencedirect.com/science/article/pii/S0300908415002850#fig1" target="_blank" rel="noopener">Fig. 1</a> and <a href="https://www.sciencedirect.com/science/article/pii/S0300908415002850#tbl1" target="_blank" rel="noopener">Table 1</a> for abbreviations and detailed structures.</p>
<p><strong>【CAPN3的转录本】</strong></p>
<p><img src="https://raw.githubusercontent.com/Cpresident/egee/master/20200205104710.png" alt=""></p>
<p>Table 2. Representative mammalian <em><a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/capn3" target="_blank" rel="noopener">CAPN3</a></em>/<em>Capn3</em> transcripts.</p>
<h2 id="Calpain3的特性"><a href="#Calpain3的特性" class="headerlink" title="Calpain3的特性"></a>Calpain3的特性</h2><ul>
<li>In mammals, CAPN3 is expressed far more strongly in skeletal muscles, especially in the fast (type II) fibers [54], than in other tissues. </li>
<li>During human development, CAPN3 appears relatively late compared to other muscle components, such as connectin/titin (TTN) and sarcoglycans (SGCs) [56], and the same is true in mice [57,58].</li>
<li><mark>CAPN3是否在发育早期才有用？</mark><br>The CAPN3 level decreases seven days after hatching, while the CAPN1 and CAPN2 levels remain constant, suggesting that the function of CAPN3 in chickens differs slightly from that inmammals [60]. </li>
<li>CAPN3 carries at least one potential nuclear-translocation signal sequence in IS2 [50],</li>
<li>In myocytes, CAPN3 exists everywheredin the cytosol; at the Zline, N2A, and M-line regions; around the sarcoplasmic reticulum (SR); and so on [7,61e66].</li>
<li>Unlike CAPN1 and CAPN2, CAPN3 produces many splicing variants (Fig. 3, Table 2). <ul>
<li><mark>IS1造成的CAPN3的不稳定？</mark>Adding IS1, but not NS or IS2, makes Lp82 as unstable as p94 [123,126].</li>
<li>In skeletal muscle, the ratio of CAPN3 variants to p94 decreases as development proceeds [57], eventually dropping to less than 1%, whereas the proportion of variants in the heart, brain, lung, and retina, and other tissues remains at 10e100% [129].</li>
<li><mark>CAPN3 splicing 在骨骼肌发育形成非常重要</mark>：Taken together with the disease phenotype of CAPN3:Dex6and Dex15-Tg mice, CAPN3 splicing variations and expression changes are likely to play important roles in skeletal-muscle development.</li>
</ul>
</li>
<li>CAPN3还有一个功能是调控SR Ca2+ release<br>In CAPN3:C129S-KI mice, the RyR and ALDOA localization and Ca2þ influx/efflux are unaffected [64]. These observations indicate that CAPN3 has a non-proteolytic functiondthat is, a structural property that is responsible for regulating SR Ca2þ release. This dual character of CAPN3 may account for the wide range of LGMD2A symptoms, at least in part.</li>
</ul>
<h2 id="CAPN3-KO-小鼠模型"><a href="#CAPN3-KO-小鼠模型" class="headerlink" title="CAPN3 KO 小鼠模型"></a>CAPN3 KO 小鼠模型</h2><ul>
<li><p>There are several mouse models for CAPN3 calpainopathy. These include CAPN3 knock-out (CAPN3-KO) mice [5,6,81]. There are also transgenic (Tg) mice expressing inactive CAPN3:C129S (CAPN3:C129S-Tg, in which C129, one of the active-site triad aars, is replaced) [82] or a splicing variant (CAPN3:Dex6-Tg and CAPN3:Dex15-Tg, in which exons 6 and 15 encoding IS1 and a part of IS2, respectively, are deleted)<br>[83]. CAPN3:C129S knock-in (CAPN3:C129S-KI) mice express CAPN3:C129S instead of wild-type (WT) CAPN3 [7].The first KO mice showed transmission-ratio distortion, which turned out to be an artifact of the introduced Pgk1 promoter [81]. All of these mouse models exhibit a myopathy phenotype with slight differences in severity, from the most benign to the most severe as follows: CAPN3:C129S-Tg &lt; CAPN3:C129SKI &lt; CAPN3-KO &lt; CAPN3:Dex15-Tg &lt; CAPN3:Dex6-Tg. These results suggest that CAPN3 and its splicing variants have complex functions in muscle.</p>
</li>
<li><p>Diabetes模型（<mark>capn3表达和脂肪化的负相关</mark>）</p>
<ul>
<li><p>The relationship between CAPN3 expression and diabetes has been examined using fat sand rats (Psammomys obesus) as an animal model of obesity [104]. </p>
</li>
<li><p>In these rats, the amount of CAPN3 in skeletal muscles is negatively correlated with fat mass and the concentration of circulating glucose, and positively correlated with carbohydrate oxidation and energy expenditure</p>
</li>
</ul>
</li>
</ul>
<h2 id="CAPN3-的固定位置"><a href="#CAPN3-的固定位置" class="headerlink" title="CAPN3 的固定位置"></a>CAPN3 的固定位置</h2><p><strong>【磷酸化的CAPN3如何固定在myofibrils上】</strong></p>
<p><img src="https://raw.githubusercontent.com/Cpresident/egee/master/20200205210628.png" alt=""></p>
<p>Fig. 8. Possible mechanisms for anchoring P-CAPN3 to the myofibrils. These mechanisms postulate the existence of an as-yet unidentified CAPN3-phosphorylating kinase, either in the cytosol (A) or myofibril-bound (B). In (A), P-CAPN3 is recruited to the myofibrils by an increased affinity for myofibrils. In (B), recruited CAPN3 is phosphorylated by myofibrilbound kinase, which may or may not increase the affinity of the molecule for the myofibrils. In either case, dissociated P-CAPN3 is rapidly dephosphorylated by phosphatases in the cytosol, increasing the accumulation of P-CAPN3 in the myofibril fraction.</p>
<ul>
<li><u>CAPN3的磷酸化: IS2</u><ul>
<li>Two phosphorylated aars have been found in human CAPN3, at S629 and S636 in IS2 (Fig. 7); the former is the major site [171]. The autolytic CAPN3 activity is slightly attenuated by the defective phosphorylation of S629 but not S636, suggesting that the <strong>autolysis is regulated in part through S629 phosphorylation</strong> [171]. Phosphorylated CAPN3 (P-CAPN3) is enriched in myofibrils rather than in the cytoplasm [163,171]. As CAPN3 directly binds TTN and ACTN2 [57,62], it is possible that CAPN3 is selectively phosphorylated in myofibrils, and this property may be related to TTN-based mechano-transduction signaling.</li>
<li><strong>Phosphorylation may also increase affinity of CAPN3 for myofibrils, causing CAPN3 to accumulate in myofibrils</strong> (Fig. 8). We found that the amounts of both CAPN3 and P-CAPN3 are reduced in LGMD2A muscle tissues [171,172].In contrast, WT CAPN3-Tg mice show almost no phenotype [83,173], suggesting that excess CAPN3 may be absorbed by myofibrils via increased phosphorylation.</li>
</ul>
</li>
</ul>
<p><img src="https://raw.githubusercontent.com/Cpresident/egee/master/20200205212834.png" alt=""></p>
<p><a href="https://www.researchgate.net/publication/51626286_Is_titin_a_%27winding_filament%27_A_new_twist_on_muscle_contraction/figures?lo=1" target="_blank" rel="noopener">Figure 1</a>. (a) Schematic of a skeletal muscle half-sarcomere illustrating the layout of titin (yellow with a red N2A segment). Each titin molecule is bound to the thin filaments (blue) in the I-band, and to the thick filaments (green) in the A-band. Note that, for simplicity, thick filaments are illustrated as double-stranded, whereas in vertebrate skeletal muscle, they are actually triple-stranded. The N2A segment is located between the proximal tandem Ig segment and the PEVK segment. (<u>b,c) Ca 2þ-dependent binding of N2A to thin filaments. (b) Resting sarcomere at slack length at low Ca 2þ concentration (pCa ¼ 9). (c) Upon Ca 2þ influx (pCa ¼ 4.5), N2A titin binds to the thin filament (blue), which shortens and stiffens the titin spring in active sarcomeres.</u></p>
<ul>
<li><p>CAPN3 binds to N2A and M-line regions of TTN [62,180], where CAPN3 is proposed to play a role in TTN-based mechanosensory transduction pathways. In stretched sarcomeres, CAPN3 accumulates at the N2A region, where a protein complex is formed in response to mechanical stimulation. The TTN I-band region extends, spring-like, as sarcomeres stretch [181,182],and thus serves as an ideal “scale” to sense sarcomere extension</p>
</li>
<li><p>In other words, inactive CAPN3 cannot rapidly access the TTN N2A regions in extended sarcomeres, and this inability may be a contributing factor in LGMD2A.</p>
</li>
<li><p>CAPN3运动后激活<br>For this physical stress response, CAPN3 protease activity is required during exercise rather than after. In addition, CAPN3 activation has been detected in muscles up to six days after exercise [7,189e191], suggesting that protease activity of CAPN3 has different roles.</p>
</li>
<li><p>锻炼前后CAPN3的位置是否需要检测？<br>Monitoring CAPN3 activity poses technical challenges; CAPN3 autolysis in skeletal muscle cells in situ cannot be detected by immunoblotting. Thus, innovative techniques are required for probing CAPN3 activity in situ to clarify its functions during and after exercise.</p>
</li>
</ul>
<h2 id="CAPN3的激活"><a href="#CAPN3的激活" class="headerlink" title="CAPN3的激活"></a>CAPN3的激活</h2><p><strong>【CAPN3的调控】</strong></p>
<p><img src="https://raw.githubusercontent.com/Cpresident/egee/master/20200205104806.png" alt=""></p>
<p>Fig. 4. A scheme of activity regulation of CAPN3. Translated full-length CAPN3 (FL-CAPN3) is iM-activated by physiological [Na+]i and [Ca2+]i (1), and autolyzes in IS1, producing nicked (but not dissociated) FL-CAPN3 (nFL-CAPN3) (2). Since nFL-CAPN3 is eM-activated and metastable, it functions as a protease (3). Then, nFL-CAPN3 is inactivated as its components (N31K and C58K) dissociate from each other (4), and/or degradative reaction predominates through autolysis in IS2 and other sites with the aid of other protease(s) (4′). N31K and C58K reconstitute the activity by iMOC1 and/or iMOC2 (5), which accelerates autolysis of FL-CAPN3 (6). Reconstituted CAPN3 by iMOC1 (iMOC-CAPN3) localizes and functions distinctly from FL/nFL-CAPN3 under alternative regulation (7), and/or is ultimately inactivated by further autolysis (7′).</p>
<ul>
<li><u>CAPN3在骨骼肌细胞通常不激活，被屏蔽</u><br>Autolytic activity of CAPN3 is suppressed in skeletal muscle cells by its binding to TTN or PLEIAD (see Section 6.2.1). However, immediately after homogenizing tissues, CAPN3 rapidly autodegrades in solutions containing Naþ, such as buffered saline</li>
<li><u>先Autolysis再activation</u><br>The activation of conventional calpains occurs almost concomitantly with their N-terminal autolysis [145e147], which is reported to precede caseinolytic activity of calpain [148] and to parallel the calpains’ autodegradation and inactivation [149].<ul>
<li>The first step of calpain activation is its intraMolecular (iM)activation upon binding Ca2þ (or, in the case of CAPN3, Naþ also). This iM-activation enables calpains to hydrolyze peptide bonds; however, steric hindrance, with the N region for example, prevents calpains from proteolyzing substrates.（<strong>The autolytic sites in IS1 are located after Y274, N296, and Y322</strong>)</li>
<li>A second step of calpain activation is that of extraMolecular (eM)-activation, which allows calpains to contact substrates. Whether autolysis is required for the eM-activation of conventional calpains is still controversial. (The titin (TTN) N2A region suppresses IS2 proteolysis and the subsequent dissociation of CAPN3 fragments.)</li>
</ul>
</li>
<li>CAPN3的内部激活可不依赖Ca2+，由NA+介导<br><u>The extreme rapidity of CAPN3 autolysis depends on IS1 and IS2</u> [65]. The exhaustiveness probably involves other protease(s), as well as ability of CAPN3 to undergo interMOlecular Complementation (iMOC) (see Section 5.1). Finally, this autolysis, which appears to be independent from Ca2þ since it is not inhibited by excess EDTA, is actually activated by Naþ [136] (see Section 4.2.2). These fascinating properties are unique among known calpains.</li>
<li><p><u>CAPN3的激活在生理离子浓度下即可</u><br>In other words, CAPN3 is activated by ca. physiological [Naþ]i and [Ca2þ]i. CAPN3 is the first (and so far, the only) Naþ-dependent intracellular enzyme.</p>
</li>
<li><p><mark>CAPN3分裂后可以再重组激活</mark><br>The amazing rapidity of CAPN3 autodegradation is thought to regulate its activity. A recent study demonstrated that de novo iMOC between the two autolytic fragments of CAPN3 reconstitutes an active core protease domain [168]</p>
<ul>
<li><u>重组是部分LGMD2A患者潜在恢复的原因?</u><br>It is currently hypothesized that iMOC may rescue a fraction of CAPN3 protease function in some cases of LGMD2A caused by distinct missense mutations in both CAPN3 alleles, and this process may alleviate symptoms [169].</li>
</ul>
</li>
</ul>
<h2 id="CAPN3的底物"><a href="#CAPN3的底物" class="headerlink" title="CAPN3的底物"></a>CAPN3的底物</h2><ul>
<li><strong>CAPN3通过α-actinin-2固定在Z-line上，底物也在Z-line上</strong><br>Potential substrates of Naþ-activated CAPN3 include tropomyosin, LIM-domain-binding protein 3, and a-actinin-3, all of which are in the Z-line [136]. CAPN3 localizes to the Z-line by binding aactinin-2 (ACTN2, sarcomeric-a-actinin) [57], indicating that Naþ dependency of CAPN3 may play a role in its functions in the Z-li</li>
</ul>
<p><strong>【CAPN3在titin上的结合位点】</strong></p>
<p><img src="https://raw.githubusercontent.com/Cpresident/egee/master/20200205232808.png" alt=""></p>
<p>Fig. 9. TTN C-terminal mutations found in TMD, and cleavage sites by calpains. A structure based on the sequence of N2A-type TTN (NP_596869; 33,423 aars, ca. 3.7 MDa, 1–1.5 μm). Red arrows indicate the C termini of sites cleaved by both CAPN3 and the conventional calpains; black arrows indicate those cleaved by calpain-2. The dotted arrow indicates a site that is probably cleaved by endogenous conventional calpains; the cleavage is suppressed by deleting mdm. The 44-kDa proteolytic fragment cleaved at the C-terminus of E33,017 is physiologically detected under a static condition [16]. Blue vertical lines indicate TMD mutations.</p>
<ul>
<li>As mentioned above, TTN serves as a large-scale CAPN3 reservoir. TTN N2A region has at least three CAPN3-binding sites [186], and binding at these sites suppresses rapid autolysis of CAPN3 [157].</li>
<li>Myospryn appears to be proteolyzed by co-expressed CAPN3 in cultured cells, and its binding to CAPN3 slightly suppresses CAPN3 autolysis [215]. Thus, this molecule may also be involved in the M-line instability caused by M10 proteolysis.</li>
<li>Biochemical studies show that CAPN3 directly interacts with Dysferlin (DYSF) [218] and with the DYSFinteracting protein AHNAK (also called desmoyokin) [219].</li>
<li>CAPN3 WT, but not C129S, has strong autolytic and proteolytic activity when expressed in cultured cells, without any additional activation process [65]. Thus, it is easy to test whether CAPN3 proteolyzes particular proteins in cultured cells. These types of studies have identified various in vitro substrates, including ALDOA [174], ANKRD23/CARP/MARP1 [241], C/EBPa [242], filamin C [156,243],IkB [244], ezrin, talin, vinexin [156], conventional calpains, calpastatin [139], b-oxidation enzyme (VLCAD) [245], TTN N2A, M- and Z-line regions [16,156,186], PIAS [246], RyR [247], AHNAK [219], spectrin [159,248], MyoD [249], and CTBP1 [208].</li>
</ul>
<h2 id="CAPN3突变导致的结果"><a href="#CAPN3突变导致的结果" class="headerlink" title="CAPN3突变导致的结果"></a>CAPN3突变导致的结果</h2><p><strong>【 CAPN3在病人的突变分布】</strong></p>
<p><img src="https://raw.githubusercontent.com/Cpresident/egee/master/20200205103727.png" alt=""></p>
<p>Fig. 2. CAPN3 missense mutations in LGMD2A patients. So far, 229 distinct missense mutations at 189 aars of CAPN3 have been reported in the Leiden Open Variation Database. The secondary structures shown are for inactive calpain-2 (1KFU).</p>
<ul>
<li><u>突变影响了不同结构域的相互作用-&gt;影响了激活离子的浓度阈值</u><br>Conventional calpains require salt bridges between the PC2 and CBSW domains and a linker between the CBSW and PEF domains for their activation and stability [117,118]. Mutations disrupting these interactions lower the [Ca2þ] required to activate calpain-2 [140,141]. </li>
<li>Based on the aar positions of most of the missense mutations that produce calpainopathies, these mutations are likely to perturb inter-domain interactions and interfere with the proper CAPN3 conformation [108].</li>
<li><strong>PC1结合Na+</strong><br>Mutations in the potential Ca2+-Binding Site(s) in PC1 (CBS-1), but not PC2 (CBS-2), compromise Naþ-dependency of CAPN3, indicating that CAPN3 CBS-1 may have evolved to bind Naþ.</li>
<li>Na+的激活需要PC1, IS1, IS2<br>Deleting both IS1 and IS2 completely abolishes Na+-dependent activation of CAPN3 without changing its Ca2+-dependent activity [136]. Thus, Naþ-dependent activity of CAPN3 requires cooperation among CBS-1, IS1, and IS2.</li>
</ul>
<h3 id="CAPN3突变的类型"><a href="#CAPN3突变的类型" class="headerlink" title="CAPN3突变的类型"></a>CAPN3突变的类型</h3><p>Although mutations that inactivate CAPN3 cause calpainopathy, not all calpainopathy-mutant CAPN3s lose their proteolytic activity [158]. There are five defective CAPN3 states: (1) completely inactive (loss of iM-activation ability), (2), inactive as to substrate proteolysis by loss of eM-activation ability, (3) inactive due to overly rapid autolytic activity or (4) improper substrate recognition, and (5) defective in non-proteolytic functions but with proteolytic activity (5).</p>
<p><strong>【Case (1): inactive mutations】</strong></p>
<p>Mutations at evolutionarily conserved aars in the CysPc domain are highly likely to inactivate CAPN3. This inactivation stabilizes CAPN3 when ectopically expressed in cultured cells [159]. However, this is not necessarily true in vivo (e.g., inactive CAPN3:V354G is lost in vivo [87]). This suggests the existence of one or more CAPN3-mutant degrading proteases that are present in skeletal muscles but not in nonmuscle cells. This is also the case with some mutations outside of CysPc. D607 in IS2 is conserved from fish to humans (see Fig. 7), and a D607A mutation attenuates the CAPN3 activity [160]. Furthermore, screening by p94-trapping, which uses a yeast two-hybrid system to detect activity-attenuated CAPN3 mutants, identified I603T, F604 L/S, V605A, and D607N [160]. Notably, a neighboring mutation, S606L, causes LGMD2A [161]. Taken together, the short, evolutionarily conserved stretch around D607 in IS2 is essential for proteolytic activity of CAPN3.</p>
<p><strong>【Case (2): mutations that disable eM-activation】</strong></p>
<p>Some IS1 mutations, including Y274A, cause defective IS1 nicking, which disables eM-activation [156]. There is an a-helix in the middle of IS1 that is disrupted by L286P [152]. Interestingly, the CysPc-only CAPN3:L286P is spontaneously activated during the purification process. This type of CAPN3 mutant may be iM- and eM-activated simultaneously, resulting in self-destruction, as described in case (3).</p>
<p><strong>【Case (3): mutations that accelerate autolysis】</strong></p>
<p>If rapid autolytic activity of CAPN3 is accelerated even further, the CAPN3 protein is lost before it can accomplish its function. This is the case with some mutations in the PEF domain, including S744G and R769Q [159]. Indeed, compared toWT, autolysis is accelerated in calpain-2 with mutations corresponding to those causing LGMD2A: T344M (T417M of CAPN3), D346E (D419E), D362K (E435K), R417W (R490W), and G423R (G496R) [162].</p>
<p>Studies using CAPN3 KO mice with transgenic expression of CAPN3 or its mutants showed that R448H and D705G proteolyzed muscle extracts faster than WT CAPN3, probably due to their inability to bind TTN [163]. This experimental system allows any CAPN3 mutant to be expressed in vivo in place of WT CAPN3. Moreover, in skeletal muscle extracts, CAPN3 showed slowed autolytic activity, and this feature might be useful for a CAPN3 autolytic assay.</p>
<p><strong>【Case (4): mutations that disrupt substrate recognition】</strong></p>
<p>In conventional calpains, CBSWand PEF are important for substrate recognition. CBSW interacts with substrate aars at the P3 and P5eP10 positions [119,120,164], and PEF is thought to assist in recognizing CaM-binding proteins as substrates [165]. Therefore, mutations may disrupt the conformation of these domains, compromising proper substrate recognition ofCAPN3. For example, R490W and R572Q show reduced, Ca2þ-dependent autolysis but can barely proteolyze calpastatin or CAPN3:C129S, which are good substrates for WT CAPN3 [159].</p>
<p><strong>【Case (5): mutations that interfere with non-proteolytic functions】</strong></p>
<p>The phenotype of CAPN3:C129S-KI mice is between that ofWT and CAPN3-KO mice [7,64], leading to the recognition of two independent CAPN3 functions: proteolytic and nonproteolytic. C129S has no proteolytic activity, but its nonproteolytic function is intact. Some mutations in regions that are not directly involved in proteolytic function (cases 1e4) may compromise only the non-proteolytic function, leaving the proteolytic function intact. This may be the case with A702V, which was found in a homozygous patient that had normal CAPN3 protein levels and activity [158].</p>
<h2 id="CAPN3的进化意义"><a href="#CAPN3的进化意义" class="headerlink" title="CAPN3的进化意义"></a>CAPN3的进化意义</h2><p><strong>【CAPN3 orthologs】</strong></p>
<p><img src="https://raw.githubusercontent.com/Cpresident/egee/master/20200205203441.png" alt=""></p>
<p>Fig. 5. Summary of CAPN3 orthologs. CAPN3s from different branches are categorized according to their molecular structure and chromosomal location; the topology of the phylogenetic tree is arbitrary. Gray indicates that the annotated sequence does not align perfectly with human CAPN3. <em>, **</em>: no exon sequences have been found that correspond to exons 15 and 16, respectively, in humans. **No canonical nuclear localization signal has been found, and the GANC gene is not identified in Ensembl (<a href="http://www.ensembl.org)" target="_blank" rel="noopener">www.ensembl.org)</a>. N/A, the genes encoding GANC and CAPN3 are found in different chromosomes. TGD, teleost genome duplication. For sequence alignment, see Ref. [11].</p>
<ul>
<li>CAPN3脊椎动物最近亲<br>As mentioned above, clear CAPN3 orthologs have been found only in vertebrates (Fig. 5).</li>
<li>一个paradox<br>Another significant feature of CAPN3 gene evolution is that CAPN3 has the most conserved orthologs among vertebrate classical calpainsdmore than either CAPN1 or CAPN2 [49] (Table 3). This is astonishing, considering that Capn2 disruption causes embryonic lethality [2,3], while Capn3 results in muscular dystrophy [5,6]. This finding suggests that vertebrates are essentially inseparable from their skeletal muscles, which constitute and maintain the whole body shape, and that CAPN3 is essentially involved in these functions. Alternatively, ancestral CAPN3 might have had functions so pivotal to cell survivaldthe regulation ofcell cycles, for instance (see Section 6.3)das to warrant the conservation of CAPN3 among vertebrates independently of its expression pattern. Further research will hopefully uncover novel CAPN3 functions.</li>
</ul>

    </article>
    <!-- license  -->
    
        <div class="license-wrapper">
            <p>Author：<a href="https://cpresident.github.io">Hua</a>
            </p><p>原文链接：<a href="https://cpresident.github.io/p/56ef.html">https://cpresident.github.io/p/56ef.html</a>
            </p><p>发表日期：<a href="https://cpresident.github.io/p/56ef.html">February 5th 2020, 10:04:00 am</a>
            </p><p>更新日期：<a href="https://cpresident.github.io/p/56ef.html">February 8th 2020, 1:53:59 pm</a>
            </p><p>版权声明：</p>
        </div>
    
    <!-- paginator  -->
    <ul class="post-paginator">
        <li class="next">
            
                <div class="nextSlogan">Next Post</div>
                <a href="/p/87c9.html" title="sarcomere形成">
                    <div class="nextTitle">sarcomere形成</div>
                </a>
            
        </li>
        <li class="previous">
            
                <div class="prevSlogan">Previous Post</div>
                <a href="/p/6d3c.html" title="Muscle MRI/ EMG mdx mice">
                    <div class="prevTitle">Muscle MRI/ EMG mdx mice</div>
                </a>
            
        </li>
    </ul>
    <!-- 评论插件 -->
    <!-- 来必力City版安装代码 -->

<!-- City版安装代码已完成 -->
    
    
    <!-- partial('_partial/comment/changyan') -->
    <!--PC版-->


    
    

    <!-- 评论 -->
</main>
            <!-- profile -->
            
        </div>
        <footer class="footer footer-unloaded">
    <!-- social  -->
    
    <div class="social">
        
    
        
            
                <a href="mailto:huatwofold@outlook.com" class="iconfont-archer email" title="email"></a>
            
        
    
        
    
        
    
        
    
        
            
                <a href="//t.me/iiijhh" class="iconfont-archer telegram" target="_blank" title="telegram"></a>
            
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
            
                <a href="/atom.xml" class="iconfont-archer rss" target="_blank" title="rss"></a>
            
        
    

    </div>
    
    <!-- powered by Hexo  -->
    <div class="copyright">
        <span id="hexo-power">Powered by <a href="https://hexo.io/" target="_blank">Hexo</a></span><span class="iconfont-archer power">&#xe635;</span><span id="theme-info">theme <a href="https://github.com/fi3ework/hexo-theme-archer" target="_blank">Archer</a></span>
    </div>
    <!-- 不蒜子  -->
    
    <div class="busuanzi-container">
    
     
    <span id="busuanzi_container_site_pv">PV: <span id="busuanzi_value_site_pv"></span> :)</span>
    
    </div>
    
</footer>
    </div>
    <!-- toc -->
    
    <div class="toc-wrapper" style=
    







top:50vh;

    >
        <div class="toc-catalog">
            <span class="iconfont-archer catalog-icon">&#xe613;</span><span>CATALOG</span>
        </div>
        
            <div id="toc-div">
        
            
            <ol class="toc"><li class="toc-item toc-level-2"><a class="toc-link" href="#Calpain3基因及其家族"><span class="toc-number">1.</span> <span class="toc-text">Calpain3基因及其家族</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#Calpain3的特性"><span class="toc-number">2.</span> <span class="toc-text">Calpain3的特性</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#CAPN3-KO-小鼠模型"><span class="toc-number">3.</span> <span class="toc-text">CAPN3 KO 小鼠模型</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#CAPN3-的固定位置"><span class="toc-number">4.</span> <span class="toc-text">CAPN3 的固定位置</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#CAPN3的激活"><span class="toc-number">5.</span> <span class="toc-text">CAPN3的激活</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#CAPN3的底物"><span class="toc-number">6.</span> <span class="toc-text">CAPN3的底物</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#CAPN3突变导致的结果"><span class="toc-number">7.</span> <span class="toc-text">CAPN3突变导致的结果</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#CAPN3突变的类型"><span class="toc-number">7.1.</span> <span class="toc-text">CAPN3突变的类型</span></a></li></ol></li><li class="toc-item toc-level-2"><a class="toc-link" href="#CAPN3的进化意义"><span class="toc-number">8.</span> <span class="toc-text">CAPN3的进化意义</span></a></li></ol>
            
        </div>
    </div>
    
    <div class="back-top iconfont-archer">&#xe639;</div>
    <div class="sidebar sidebar-hide">
    <ul class="sidebar-tabs sidebar-tabs-active-0">
        <li class="sidebar-tab-archives"><span class="iconfont-archer">&#xe67d;</span><span class="tab-name">Archive</span></li>
        <li class="sidebar-tab-tags"><span class="iconfont-archer">&#xe61b;</span><span class="tab-name">Tag</span></li>
        <li class="sidebar-tab-categories"><span class="iconfont-archer">&#xe666;</span><span class="tab-name">Cate</span></li>
    </ul>
    <div class="sidebar-content sidebar-content-show-archive">
          <div class="sidebar-panel-archives">
    <!-- 在ejs中将archive按照时间排序 -->
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    <div class="total-and-search">
        <div class="total-archive">
        Total : 22
        </div>
        <!-- search  -->
        
            <div class="site-search popup-trigger">
                <span class="iconfont-archer search-icon">&#xe627;</span>
            </div>
        
    </div>
    
    <div class="post-archive">
    
    
    
    
    <div class="archive-year"> 2020 </div>
    <ul class="year-list">
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">02/08</span><a class="archive-post-title" href="/p/12a4.html">Introduction to miRNA</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">02/07</span><a class="archive-post-title" href="/p/6b84.html">miRNA in RTX MG</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">02/06</span><a class="archive-post-title" href="/p/87c9.html">sarcomere形成</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">02/05</span><a class="archive-post-title" href="/p/56ef.html">Calapin3详析</a>
        </li>
    
    
    
    
    
        </ul>
    
    <div class="archive-year"> 2019 </div>
    <ul class="year-list">
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">08/13</span><a class="archive-post-title" href="/p/6d3c.html">Muscle MRI/ EMG mdx mice</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">06/14</span><a class="archive-post-title" href="/p/6dce.html">MG相关、B细胞相关mirco RNA</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">06/09</span><a class="archive-post-title" href="/p/d463.html">MG相关micro RNA（2019-06-09）</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">05/18</span><a class="archive-post-title" href="/p/9b9f.html">重症肌无力国内国外文献分析报告 （2019-5-18）</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">04/27</span><a class="archive-post-title" href="/p/a0b7.html">神经免疫类与外周血T淋巴细胞研究套路</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">04/17</span><a class="archive-post-title" href="/p/ef21.html">TCM和TEM</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">03/18</span><a class="archive-post-title" href="/p/94c7.html">HLA in Myasthenia gravis【稿4】报告</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">03/14</span><a class="archive-post-title" href="/p/5848.html">HLA in Myasthenia gravis【V4.5.1】报告</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">03/12</span><a class="archive-post-title" href="/p/8e00.html">HLA in Myasthenia gravis【V4.5】报告</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">02/08</span><a class="archive-post-title" href="/p/3563.html">HLA in Myasthenia gravis【稿3】插图及表格</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">02/04</span><a class="archive-post-title" href="/p/35ac.html">GCSRT课程</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">02/04</span><a class="archive-post-title" href="/p/4e76.html">HLA in Myasthenia gravis【稿3】报告</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">02/02</span><a class="archive-post-title" href="/p/b5c6.html">Treg cell的发生发展</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">01/22</span><a class="archive-post-title" href="/p/e83a.html">HLA in Myasthenia gravis 【稿2】 报告</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">01/18</span><a class="archive-post-title" href="/p/2d4a.html">HLA in Myasthenia gravis 【Meta可行性分析】</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">01/15</span><a class="archive-post-title" href="/p/ac35.html">HLA in Myasthenia gravis 【稿1】 报告</a>
        </li>
    
    
    
    
    
        </ul>
    
    <div class="archive-year"> 2018 </div>
    <ul class="year-list">
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">12/31</span><a class="archive-post-title" href="/p/3d25.html">HLA and Myasthenia gravis【R环境下Biblioshiny分析报告】</a>
        </li>
    
    
        <li class="archive-post-item">
            <span class="archive-post-date">12/28</span><a class="archive-post-title" href="/p/e9d9.html">BCR is the New Antibody, TCR is still a T Cell Receptor</a>
        </li>
    
    </ul></div>
  </div>
        <div class="sidebar-panel-tags">
    <div class="sidebar-tags-name">
    
        <span class="sidebar-tag-name" data-tags="MG"><span class="iconfont-archer">&#xe606;</span>MG</span>
    
        <span class="sidebar-tag-name" data-tags="LGMD"><span class="iconfont-archer">&#xe606;</span>LGMD</span>
    
        <span class="sidebar-tag-name" data-tags="miRNA"><span class="iconfont-archer">&#xe606;</span>miRNA</span>
    
        <span class="sidebar-tag-name" data-tags="RTX"><span class="iconfont-archer">&#xe606;</span>RTX</span>
    
    </div>
    <div class="iconfont-archer sidebar-tags-empty">&#xe678;</div>
    <div class="tag-load-fail" style="display: none; color: #ccc; font-size: 0.6rem;">
    缺失模块。<br>
    1、请确保node版本大于6.2<br>
    2、在博客根目录（注意不是archer根目录）执行以下命令：<br>
    <span style="color: #f75357; font-size: 1rem; line-height: 2rem;">npm i hexo-generator-json-content --save</span><br>
    3、在根目录_config.yml里添加配置：
    <pre style="color: #787878; font-size: 0.6rem;">
jsonContent:
  meta: false
  pages: false
  posts:
    title: true
    date: true
    path: true
    text: false
    raw: false
    content: false
    slug: false
    updated: false
    comments: false
    link: false
    permalink: false
    excerpt: false
    categories: true
    tags: true</pre>
    </div> 
    <div class="sidebar-tags-list"></div>
</div>
        <div class="sidebar-panel-categories">
    <div class="sidebar-categories-name">
    
        <span class="sidebar-category-name" data-categories="PP"><span class="iconfont-archer">&#xe60a;</span>PP</span>
    
        <span class="sidebar-category-name" data-categories="OA"><span class="iconfont-archer">&#xe60a;</span>OA</span>
    
    </div>
    <div class="iconfont-archer sidebar-categories-empty">&#xe678;</div>
    <div class="sidebar-categories-list"></div>
</div>
    </div>
</div> 
    <script>
    var siteMeta = {
        root: "/",
        author: "Hua"
    }
</script>
    <!-- CDN failover -->
    <script src="https://cdn.jsdelivr.net/npm/jquery@3.3.1/dist/jquery.min.js"></script>
    <script type="text/javascript">
        if (typeof window.$ === 'undefined')
        {
            console.warn('jquery load from jsdelivr failed, will load local script')
            document.write('<script src="/lib/jquery.min.js">\x3C/script>')
        }
    </script>
    <script src="/scripts/main.js"></script>
    <!-- algolia -->
    
        <div class="site-search">
  <div class="algolia-popup popup">
    <div class="algolia-search">
      <div class="algolia-search-input-icon">
        <i class="fa fa-search"></i>
      </div>
      <div class="algolia-search-input" id="algolia-search-input"></div>
    </div>

    <div class="algolia-results">
      <div id="algolia-stats"></div>
      <div id="algolia-hits"></div>
      <div id="algolia-pagination" class="algolia-pagination"></div>
    </div>

    <span class="popup-btn-close">
      <i class="iconfont-archer">&#xe609;</i>
    </span>
  </div>
</div>
        <script src="https://cdn.jsdelivr.net/npm/instantsearch.js@2.8.0/dist/instantsearch.min.js" defer></script>
        <script src="/scripts/search.js" defer></script>
    
    <!-- busuanzi  -->
    
    <script async src="//busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
    
    <!-- CNZZ  -->
    
    </div>
    <!-- async load share.js -->
    
        <script src="/scripts/share.js" async></script>    
     
    </body>
</html>


